### ISSUE TITLE
voiceover screen reader support MacOS X

### ISSUE THREAD

**Issue by gkearney** [atom/atom#7460]
Atom has no VoiceOver screen reader support on MacOS X. This means it is unusable by the blind and print disabled. VO is part of OS X and should be supported. 

Tested on MacOS X 10.10

**Comment by batjko** [comment]
I would actually remove the MacOS X qualification here, as this is probably a valid feature request for any OS.

**Comment by gkearney** [comment]
That may be true but I have only tested it on MacOS X and can not speak to other environments

**Comment by mnquintana** [comment]
Thanks for the suggestion! This seems like it's a duplicate of https://github.com/atom/atom/issues/1691 – does that issue cover what you're looking for?

**Comment by gkearney** [comment]
Yes my issue is the same as #1691


Commonwealth Braille & Talking Book Cooperative
Greg Kearney, General Manager
#320, 185-911 Yates Street
Victoria, BC V8V 4Y9
CANADA
Email: info@cbtbc.org

U.S. Address
21908 Almaden Av.
Cupertino, CA 95014
UNITED STATES
Email: gkearney@gmail.com

**Comment by mnquintana** [comment]
Thanks for the update! I'm going to close this in favor of https://github.com/atom/atom/issues/1691 to continue the discussion there.
